src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -617,9 +743,9 @@ class AddItemsPage extends Component {
 
           return {
             ...val,
+            quantityOnHand: '',
             disabled: true,
             quantityRequested: qtyRequested >= 0 ? qtyRequested : 0,
-            quantityOnHand: isPullType ? quantityOnHand : undefined,
```

Comment:
Why did you make this change?

Reply:
It was said that the qoh shall be blank, would it be blank in any circumstances if I had left it as it was?